specifications/api/swagger.yaml

```diff
@@ -2759,7 +2759,7 @@ components:
         mint: *ApiAssetMintBurn
         burn: *ApiAssetMintBurn
         validity_interval: *ApiValidityIntervalExplicit
-        script_integrity: *scriptIntegrityHash 
+        script_integrity: *scriptIntegrityHash
```

Comment:
I wonder who is leaving these stray trailing spaces everywhere. I also deleted the same space in one of my PRs. 😄